tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/Deduplication/DeduplicationTests.cs

```diff
@@ -23,6 +23,8 @@ public DeduplicationTests(ITestOutputHelper output)
         : base("Deduplication", output)
     {
         SetServiceVersion("1.0.0");
+
+        SetEnvironmentVariable("DD_TRACE_DEBUG", "1");
```

Comment:
You shouldn't add `DD_TRACE_DEBUG` to integration tests in general (it's fine for testing, but we made it a vague policy not to add it by default)

Reply:
True, I was troubleshooting that test in CI and the env var slipped by... I'm removing it

Reply:
We had a similar problem on our unit tests with the GlobalSettings enabling debug mode.. wondered if we could have some kind of analyzer on our tests for that 🤔

Reply:
It's a nice idea, though it's not particularly easy to do! 😅